packages/manager/src/features/OneClickApps/AppDetailDrawer.tsx

```diff
@@ -117,13 +114,17 @@ export const AppDetailDrawer: React.FunctionComponent<Props> = (props) => {
     <Drawer
       anchor="right"
       classes={{ paper: classes.paper }}
+      data-qa-drawer
+      data-testid="drawer"
```

Comment:
```suggestion
      data-qa-drawer
      data-testid="drawer"
```
Nice catch. Curious what's special about this drawer that we need to use the MUI component and re-implement these attributes and the close button instead of using the component in `src/components/Drawer.tsx`

Reply:
Yeah i thought about moving it to our drawer but it would have expanded the scope too much cause I would have had to add props to our Drawer etc. for another time 😄